beast/physicsmodel/stars/ezmist/simpletable.py

```diff
@@ -71,13 +72,13 @@
 if PY3:
     iteritems = operator.methodcaller('items')
     itervalues = operator.methodcaller('values')
-    basestring = (str, bytes)
+    str = (str, bytes)
```

Comment:
str -> strtype as in simpletable.py in ezpadova throughout this file, or one could simply duplicate ezpadova's simpletable.py here.

Reply:
Here the variable currently named `str`, which is being set equal to `(str, bytes)`, should be renamed strtype throughout the file.  Not sure one can make a variable named `str` since it is a built-in type.  This substitution would make ezmist/simpletable.py consistent with the edits made to ezpadova/simpletable.py (in fact, the files should be identical in the end).

Reply:
I agree with Cliff here, you should not redefine the str type variable, esp. by a tuple containing itself.

Reply:
Good point.  In fact, having two copies of the same file is not good either.  So I moved the simpletable.py from ezpadova to the parent directory and have changed both padova.py and mist.py to import from this single file.  Easier to maintain if we have just one version of this file.